resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,69 @@
+@extends('layouts.app')
```

Comment:
why we have used blade templates on backend?  This should be front end components. The same for others below.

Reply:
deleted